src/components/Modal.tsx

```diff
@@ -1,8 +1,9 @@
 import assign = require('lodash/assign');
 import * as React from 'react';
+import { ModalProps } from 'rendition';
```

Comment:
Sweet, is this how we can import typings from our definition without redefining them?

Reply:
I think so, everything seems to work until now (also tested the integration with the UI).